system-addon/content-src/components/TopSites/TopSiteForm.jsx

```diff
@@ -68,6 +120,22 @@ export class TopSiteForm extends React.PureComponent {
     }
   }
 
+  onPreviewButtonClick(event) {
+    event.preventDefault();
+    if (this.validateForm()) {
+      this.setState({pendingScreenshotUpdate: true});
```

Comment:
Any reason to have this in the component state vs setting true in redux probably by PREVIEW_REQUEST and cleared on SCREENSHOT_PREVIEW/PREVIEW_FAILED?